builtin/files/stack-templates/control-plane.json.tmpl

```diff
@@ -219,12 +219,14 @@
             "ToPort": 22
           },
           {{end -}}
+          {{ if .openICMP.Enabled -}}
           {
             "CidrIp": "0.0.0.0/0",
             "FromPort": -1,
             "IpProtocol": "icmp",
             "ToPort": -1
           }
```

Comment:
I think this might be your issue. Now that this is optional (`{{ if .... }}`) the block above is going to leave a trailing `,` if this is disabled, like this.

```
....
    "ToPort": 22
  },
]
```

We can't have that trailing slash if `openICMP` is disabled.